src/main/kotlin/org/jitsi/nlj/RtpReceiverImpl.kt

```diff
@@ -173,6 +179,7 @@ class RtpReceiverImpl @JvmOverloads constructor(
                                 predicate = PacketPredicate { it is AudioRtpPacket }
                                 path = pipeline {
                                     node(silenceDiscarder.rtpNode)
+                                    node(audioBitrateCalculator)
```

Comment:
You have the bitrate calculator (which handles "is sending") after the silence discarder.  Perhaps switch these?

Reply:
This is intentional, because browsers send packets while muted

Reply:
Oh, they do? Okay.

Reply:
Just tested: chrome sends 50pps, ~13kbps